query.rb

```diff
@@ -99,9 +98,8 @@
           puts (barcode.nil? ? '(General)' : barcode).bold
           processor.warning_query[barcode].each do |h|
             puts h[:stage].brown
-            h[:warnings].each do |warn|
-              puts "  #{'File'.bold}\t#{warn.path}" unless warn.path.nil?
-              puts "  #{'Error'.bold}\t#{warn.description.italic}"
+            h[:warnings].each do |err|
```

Comment:
@moseshll what does the h stand for?

Reply:
Hash. I tend to hyper-abbreviate block params by default but I think it's a bad habit. Would `query_hash` scan better? Or just `hash`?

Reply:
I think it's fine as is now that I know.